content/uiux/concepts/problem-statement/problem-statement.md

```diff
@@ -0,0 +1,18 @@
+---
+Title: 'Problem Statement'
+Description: 'A problem statement succinctly describes what problem a design will attempt to solve. It is a standard output of the “define” stage of the design thinking process.'
+Subjects:
+  - 'Web Design'
+Tags:
+  - 'UI'
+  - 'UX'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+A **problem statement** succinctly describes what problem a design will attempt to solve. It is a standard output of the “define” stage of the [design thinking process](https://www.codecademy.com/resources/docs/uiux/design-thinking).  
```

Comment:
[suggestion] I think adding an example of a problem statement would help readers gain a better understanding of how it looks. 
```suggestion
A **problem statement** succinctly describes what problem a design will attempt to solve. It is a standard output of the “define” stage of the [design thinking process](https://www.codecademy.com/resources/docs/uiux/design-thinking).  

### Example

```